src/additional_services/explorer.star

```diff
@@ -0,0 +1,51 @@
+databases_package = import_module("../../databases.star")
+
+
+def apply_default(args):
+    return {
+        "image": "sotazklabs/stark_compass_explorer:v0.2.38",
+        "name": "madara-explorer",
+        "port": "4000"
+    } | args
+
+def run(plan, args):
+    args = apply_default(args)
+
+    ports = {}
+    ports["http"] = PortSpec(
+        number=args["port"],
+        application_protocol="http",
+        wait=None
+    )
+
+    name = args["name"] + args["deployment_suffix"],
+
+    madara_service = plan.get_service(name="madara" + args["deployment_suffix"])
+    madara_url = "http://{}:9944".format(
+        madara_service.ip_address, madara_service.ports["rpc"].number
+    )
+
+    postgres_service = plan.get_service(name="postgres" + args["deployment_suffix"])
+    postgres_url = "postgresql://madara_explorer_user:redacted@{}:5432/madara_explorer_db".format(
```

Comment:
?

Reply:
I see that the database configuration includes a database for the Madara Explorer. I will try to retrieve this configuration from the Postgres service instead of hardcoding it.